nautobot/ipam/migrations/0022_aggregate_to_prefix_data_migration.py

```diff
@@ -0,0 +1,235 @@
+    for tag in Tag.objects.filter(content_types=aggregate_ct):
+        tag.content_types.add(prefix_ct)
+
+    # migrate Aggregate ObjectPermissions to Prefix
+    for object_permission in ObjectPermission.objects.filter(object_types=aggregate_ct):
+        object_permission.object_types.add(prefix_ct)
+
+    # migrate Aggregate CustomFields to Prefix
+    for custom_field in CustomField.objects.filter(content_types=aggregate_ct):
+        custom_field.content_types.add(prefix_ct)
+
+    # migrate Aggregate CustomLinks to Prefix
+    CustomLink.objects.filter(content_type=aggregate_ct).update(content_type=prefix_ct)
+
+    # migrate Aggregate ComputedFields to Prefix
+    ComputedField.objects.filter(content_type=aggregate_ct).update(content_type=prefix_ct)
+
+    # migrate Aggregate Relationships to Prefix
+    Relationship.objects.filter(source_type=aggregate_ct).update(source_type=prefix_ct)
+    Relationship.objects.filter(destination_type=aggregate_ct).update(destination_type=prefix_ct)
+
+    # migrate individual aggregates to prefixes
+    for instance in Aggregate.objects.all():
+        if Prefix.objects.filter(
+            network=instance.network,
+            prefix_length=instance.prefix_length,
+            vrf__isnull=True,
+        ).exists():
+            prefix = Prefix.objects.get(network=instance.network, prefix_length=instance.prefix_length)
+            mismatches.update(_migrate_aggregate_to_existing_prefix(instance, prefix))
+
+        else:
+            prefix = Prefix.objects.create(
+                id=instance.id,
+                broadcast=instance.broadcast,
+                date_allocated=_convert_date_to_datetime(instance.date_added),
+                description=instance.description,
+                network=instance.network,
+                prefix_length=instance.prefix_length,
+                rir=instance.rir,
+                status=prefix_default_status,
+                tenant=instance.tenant,
+                type=PrefixTypeChoices.TYPE_CONTAINER,
+            )
+
+        # use `migrated_to_prefix` ForeignKey reference on Aggregate to assist with app migrations
+        instance.migrated_to_prefix = prefix
+        instance.save()
+
+        # migrate all notes assigned to this aggregate to the new prefix
+        _migrate_aggregate_notes_to_prefix(apps, instance, aggregate_ct, prefix_ct)
+
+        # migrate all object changes related to this aggregate to the new prefix
+        _migrate_aggregate_object_changes_to_prefix(apps, instance, aggregate_ct, prefix_ct)
+
+        # migrate all tags related to this aggregate to the new prefix
+        _migrate_aggregate_tags_to_prefix(apps, instance, aggregate_ct, prefix_ct)
+
+        # migrate custom fields on this aggregate to the new prefix
+        mismatches.update(_migrate_aggregate_custom_fields_to_prefix(instance))
+
+        # migrate all relationships related to this aggregate to the new prefix
```

Comment:
This might be a bit more accurate? Same with the function name
```suggestion
        # migrate all relationship associations related to this aggregate to the new prefix
```